FusionIIIT/applications/ps2/models.py

```diff
@@ -21,4 +21,16 @@ class StockEntry(models.Model):
     receiver_name = models.CharField(max_length=50, blank=False)
 
     class Meta:
-        db_table = 'ps2_stocks'
\ No newline at end of file
+        db_table = 'ps2_stocks'
+
+class TransferEntry(models.Model):
+    Item_id = models.IntegerField(blank=False)
+    From_department = models.CharField(max_length=50, blank=False)
+    From_location = models.IntegerField(blank=True, null=True)
```

Comment:
Why is this an Integerfield?

Reply:
Currently ID is auto generated from Django itself and it is integer